### PR TITLE
:pencil: Updates contact list and fixes link to membership

### PR DESCRIPTION
**Pull request by jefftriplett**
Updates:
- Fixed link to our charter/list of members
- Update our ED role/contact info
- Removed two members who aren't on the board

### CODE REVIEW

File: reporting-procedures.md [hunk 1]
```diff
@@ -10,22 +10,16 @@ If you find that you need to make a report, and you cannot find the appropriate
 
 *General PSF reporting procedure:*
 
-The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [LINK](). 
+The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are). 
 
 In the event of a [conflict of interest](), you may directly contact any of the lead incident responders:
```
Comment (drewbrew) [inline reporting-procedures.md:15]: should that link resolve somewhere?

File: reporting-procedures.md [hunk 2]
```diff
@@ -10,22 +10,16 @@ If you find that you need to make a report, and you cannot find the appropriate
...
-      * <chair@python.org>      
+      * <deb@python.org>
   * Van Lindberg
       * Python Software Foundation - General Counsel
       * <van@python.org>
-  * Christopher Neugebauer
-      * Python Software Foundation Board, liason to Code of Conduct Committee
-      * <chrisjrn@python.org>
 
 You can also contact any member of the Python Software Foundation [Board of Directors]() to make a report. Certain gatherings and online fora, will have additional lead responders for the purpose of helping address incidents in particular PSF-affiliated spaces.
```
Comment (drewbrew) [inline reporting-procedures.md]: same here

File: reporting-procedures.md (hunk 1 above)
Comment (treyhunner) [reply reporting-procedures.md:15]: Maybe here? https://en.wikipedia.org/wiki/Conflict_of_interest

File: reporting-procedures.md [hunk 3]
```diff
@@ -10,22 +10,16 @@ If you find that you need to make a report, and you cannot find the appropriate
 
 *General PSF reporting procedure:*
 
-The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [LINK](). 
+The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are).
```
Comment (treyhunner) [inline reporting-procedures.md]: "this alias is listed at" -> "this alias are listed at"

File: reporting-procedures.md (hunk 2 above)
Comment (treyhunner) [reply reporting-procedures.md]: We could link to this: https://www.python.org/psf/board/#id1

I don't trust the `#id` anchor though...

File: reporting-procedures.md (hunk 3 above)
Comment (jefftriplett) [reply reporting-procedures.md]:
```suggestion
The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias are listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are). 
```

File: reporting-procedures.md [hunk 4]
```diff
@@ -10,22 +10,16 @@ If you find that you need to make a report, and you cannot find the appropriate
...
-      * <chair@python.org>      
+      * <deb@python.org>
   * Van Lindberg
       * Python Software Foundation - General Counsel
       * <van@python.org>
-  * Christopher Neugebauer
-      * Python Software Foundation Board, liason to Code of Conduct Committee
-      * <chrisjrn@python.org>
 
 You can also contact any member of the Python Software Foundation [Board of Directors]() to make a report. Certain gatherings and online fora, will have additional lead responders for the purpose of helping address incidents in particular PSF-affiliated spaces.
```
Comment (jefftriplett) [inline reporting-procedures.md]:
```suggestion
You can also contact any member of the Python Software Foundation [Board of Directors](https://www.python.org/psf/board/#id1) to make a report. Certain gatherings and online fora will have additional lead responders for the purpose of helping address incidents in particular PSF-affiliated spaces. 
```

File: reporting-procedures.md (hunk 1 above)
Comment (jefftriplett) [reply reporting-procedures.md:15]: The link is resolving for me with incognito mode so I'm not seeing this.

Comment (treyhunner) [reply reporting-procedures.md:15]: The link goes one level up (when I'm at https://github.com/psf/community-code-of-conduct/blob/ee92c25ce2680ae99bd3437b33d73a8e6c105460/reporting-procedures.md it goes to https://github.com/psf/community-code-of-conduct/blob/ee92c25ce2680ae99bd3437b33d73a8e6c105460/ for example) for me.

Comment (drewbrew) [reply reporting-procedures.md:15]: Right now GitHub resolves that `()` to `https://github.com/psf/community-code-of-conduct/blob/update-contacts` which is a 404

File: reporting-procedures.md [hunk 5]
```diff
@@ -10,24 +10,18 @@ If you find that you need to make a report, and you cannot find the appropriate
 
 *General PSF reporting procedure:*
 
-The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [LINK](). 
+The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias are listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are).
```
Comment (terezaif) [inline reporting-procedures.md]:
```suggestion
The best way to contact the PSF Code of Conduct working group is by email at **<[conduct-wg@python.org](mailto:conduct-wg@python.org)>**. The members of the Code of Conduct Working Group who monitor this alias are listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are). 
```

File: reporting-procedures.md (hunk 1 above)
Comment (drewbrew) [reply reporting-procedures.md:15]: https://github.com/psf/community-code-of-conduct/pull/11 separate PR for this one

**Pull request merged**